test/controllers/user.test.js

```diff
@@ -213,4 +214,31 @@ describe('Users Controller', () => {
       });
     });
   });
+
+  describe('/POST users/sessions/invalidate_all', () => {
+    describe('when using valid parameters', () => {
+      it('should invalidate all session tokens', done => {
+        factory.create('User', { password: 'QSShBtjP' }).then(user => {
+          request
+            .post('/api/v1/users/sessions')
+            .send({
+              mail: user.mail,
+              password: '$2b$10$G4b27Ilbv5Jmg8IrtWzjUuiY3zD2wvG9OuWlXEbg60F5Xy8s1Z12u'
+            })
```

Comment:
Avoid test two endpoints together, generate fake data directly (valid tokens) and test the endpoint/middleware